src/verify.h

```diff
@@ -0,0 +1,37 @@
+#ifndef VERIFY_H
+#define VERIFY_H
+#include <stdint.h>
+
+// Store the return value of the code.  Like command line tools, a value >=1
+// will be a failure and == 0 will be success.
+typedef unsigned int return_t;
```

Comment:
perhaps `VF_result_t` just to avoid potential name conflicts?

Reply:
done